test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -70,12 +66,13 @@ class StructuredFormDataBuilderSpec extends Spec {
         "field-year"  -> "3"
       ),
       objectStructure(
-        "field" -> objectStructure(
-          "day"   -> textNode("1"),
-          "month" -> textNode("2"),
-          "year"  -> textNode("3")
-        )
-      )
+        field(
+          "field",
+          objectStructure(
+            field("day", textNode("1"), Map.empty),
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.